tests/integrated/test-command-args/runtest

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+# Compile command-args
+make || exit 1
+
+# Remove directories which may be left from previous run
+rm -r data
+rm -r test
+
+# Run without any directory, check that an exception is thrown
+echo "=========== Running without input directory =============="
+./command-args 2> stderr.log
+if ! grep -i "exception thrown" stderr.log > output.log ; then
```

Comment:
I would prefer to check the exit status.
The "exception thrown" may be added for a strange reason at some point in the future to the normal output.